polyphemus/config.yml.template

```diff
@@ -1,7 +1,11 @@
 :development:
   :magma:
-    :host: https://magma.ucsf.edu
+    :host: https://magma.development.local
+  :mvir1:
+    :token: my-token # create a long lived token with janus
```

Comment:
I imagine configuring a mvir1 project specific long lived token here for now until we implement proper impersonation via s2s authentication.